### PR TITLE
About Page Design Implementation #29

### PR DESCRIPTION
**Pull request by Anniekobia**
## What is the Purpose?
Implementing the About page design  

## What was the approach?
As per design guidelines

## Are there any concerns to addressed further before or after merging this PR?

- There are nested RelativeLayouts I used to avoid too many Constraints. Feedback on the same would be appreciated.

- There are additions to be done:
    
         1. Organizers details/bio page design
         2. Include other organizers logos once available on the design link  

## Mentions?
@michaelbukachi @wangerekaharun 

## Issue(s) affected? 
#29

### CODE REVIEW

**Comment by wangerekaharun** [comment]
[About Module Design](https://trello.com/c/wxcutNzv/12-about-module-design)

**Review by wangerekaharun — COMMENTED** [review]
Great work on the PR @Anniekobia . I particularly like the fact that we have some constraint layout there.

Here are a couple of things i to polish on before we merge this:
- For members its better to use recyclerview since they are not fixed. Rather let it be dynamic.
- Also add the TeamMember Details Fragment and its design too. Incase you need help     on this kindly say so that we can speed up on the design

Overall everything else is perfect!

**Comment by michaelbukachi** [comment]
Awesome work! I can help out with the "too many constraints" part once you've finished.

**Comment by Anniekobia** [comment]
Alright, thanks for the feedback. For the TeamMemberDetails design I think I can tweak the SpeakerDetails since its more or less the same design but with a few additions

**Comment by Anniekobia** [comment]
I have pushed the changes for the TeamMemberDetails design. I included it in the about module. Also placed its design on the sessions tab for illustration. We can remove it later once everything is okay. Still working on the recyclerview for the team member profiles

**Comment by Anniekobia** [comment]
I have replaced the members views on the About page to use a recyclerview in my latest commit. This covers all the changes requested for the PR. Kindly look at it and comment @wangerekaharun @michaelbukachi

File: features/about/src/main/res/layout/fragment_about.xml [hunk 1]
```diff
@@ -1,11 +1,158 @@
...
+            android:text="@string/about_heading"
+            android:textColor="@color/colorPurpleHeading"
+            android:textSize="@dimen/about_heading_text_size"
+            app:layout_constraintBottom_toTopOf="@+id/txt_about_content"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/about_page_image" />
+
+        <TextView
+            android:id="@+id/txt_about_content"
+            android:layout_width="wrap_content"
```
Comment (michaelbukachi) [inline features/about/src/main/res/layout/fragment_about.xml]: Change `android:layout_width` to `0dp` so that text adapts to the constraints.

**Review by michaelbukachi — APPROVED** [review]

**Review by wangerekaharun — APPROVED** [review]

**Review by MuindiStephen — COMMENTED** [review]
LGTM

**Pull request merged**